parsl/executors/workqueue/executor.py

```diff
@@ -42,6 +44,13 @@
 
 logger = logging.getLogger(__name__)
 
+package_analyze_script = os.path.join(
```

Comment:
Let's add these as entry_points in `parsl/setup.py`. It's not a good idea to mess with pythonpaths.